meinberlin/apps/budgeting/rules.py

```diff
@@ -20,9 +21,13 @@
 
 rules.add_perm(
     'meinberlin_budgeting.rate_proposal',
-    module_predicates.is_allowed_rate_item
+    module_predicates.is_allowed_rate_item | is_allowed_support_item
```

Comment:
I think you already explained it, but could you do that again please?
Why do we need `is_allowed_support` here, but not `is_allowed_rate` in the support proposal perm?

Reply:
@fuzzylogic2000 We need it here for the rating api, that we are using and that checks for rate_proposal on POST.